src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -370,6 +370,39 @@ module Types = struct
               Stringable.Ledger_hash.to_base58_check
               @@ Staged_ledger_hash.ledger_hash staged_ledger_hash ) ] )
 
+  let natural_number_desc field_name num_bits =
+    Core.sprintf
+      !"String encoded natural number of the %s (%i bits)"
+      field_name num_bits
+
+  let consensus_state =
+    let open Consensus.Data.Consensus_state in
+    obj "ConsensusState" ~fields:(fun _ ->
+        [ field "blockchainLength" ~typ:(non_null string)
+            ~doc:
```

Comment:
can we make the doc for these more descriptive? Assuming you move the number stuff into `typ`, for this we can say:

```
blockchainLength: "Length of the blockchain at this block"
slot: "Slot in which this block was created"
epoch: "Epoch in which this block was created"
totalCurrency: "Total currency in circulation at this block"
```